notesapi/v1/tests/test_views.py

```diff
@@ -65,6 +66,16 @@ def _create_annotation(self, **kwargs):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         return response.data.copy()
 
+    def _create_annotation_without_assert(self, **kwargs):
+        """
+        Create annotation but do not check for 201
+        """
+        opts = self.payload.copy()
+        opts.update(kwargs)
+        url = reverse('api:v1:annotations')
+        response = self.client.post(url, opts, format='json')
+        return response
```

Comment:
no if I add a third argument I will have to change all calls, if I send the flag via kwargs it will change the payload for post.